src/main/main.c

```diff
@@ -109,7 +109,6 @@ void
 parse_options_again(void)
 {
 	if (!init_o) {
-		load_config();
```

Comment:
Ok, but why did you move load_config() ?

Reply:
It wasn't necessary after all. I was working on a previous version of the code where the configuration loading took place in a process and the mouse opening in another.